apps/wallet/src/ui/app/pages/accounts/manage/AccountGroup.tsx

```diff
@@ -48,8 +50,10 @@ function AccountFooter({ accountID }: { accountID: string }) {
 		<div className="flex flex-shrink-0 w-full">
 			<div className="flex gap-3">
 				<div className="w-4" />
-				<FooterLink to={`/accounts/edit/${accountID}`}>Edit Nickname</FooterLink>
-				<FooterLink to="/remove">Remove</FooterLink>
+				<NicknameDialog accountID={accountID} trigger={<FooterLink>Edit Nickname</FooterLink>} />
+				<FooterLink to="/remove">
+					<div className="pt-[1px]">Remove</div>
```

Comment:
we can avoid this `pt-[1px]` class by adding an `items-center` on line 51 where we have `flex gap-3`